Formula/mariadb.rb

```diff
@@ -39,14 +34,6 @@ class Mariadb < Formula
     :because => "both install plugins"
 
   def install
-    if build.devel?
```

Comment:
Well 10.1.23 got released after 10.2.5 and contains fixes that were not yet in 10.2.5. :)